### PR TITLE
FIX Add extraEmptyValues to TreedropdownField

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1616

Test setup provided here https://github.com/silverstripe/silverstripe-admin/issues/1616#issuecomment-1801063932

### CODE REVIEW

File: src/Forms/TreeDropdownField.php [hunk 1]
```diff
@@ -984,4 +989,16 @@ public function setShowSelectedPath($showSelectedPath)
         $this->showSelectedPath = $showSelectedPath;
         return $this;
     }
+
+    /**
+     * @return array
+     */
+    public function getSchemaValidation()
```
Comment (sabina-talipova) [inline src/Forms/TreeDropdownField.php:996]: Could you, please, add unit test for this method.

**Review by sabina-talipova — CHANGES_REQUESTED** [review]

File: src/Forms/TreeDropdownField.php (hunk 1 above)
Comment (emteknetnz) [reply src/Forms/TreeDropdownField.php:996]: Done

Note - the parent card on this is a SPIKE so don't merge this just yet, need to agree with team first that we're happy with the approach before we merge anything

**Review by sabina-talipova — APPROVED** [review]
LGTM

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Overall seems okay to me - though as a bugfix that affects CMS 4 and doesn't add any new API, this should target 4.13, shouldn't it?

**Comment by emteknetnz** [comment]
Closed in favor of https://github.com/silverstripe/silverstripe-framework/pull/11068

**Pull request closed without merging**